frontend/app/component/form/data_entry/candidates_votes/CandidatesVotesForm.tsx

```diff
@@ -92,21 +92,27 @@ export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
     }
   }, [isSaved, errors, warnings]);
 
+  const [missingTotalError, setMissingTotalError] = React.useState(false);
   const [warningsWarning, setWarningsWarning] = React.useState(false);
 
   const handleSubmit = (event: React.FormEvent<CandidatesVotesFormElement>) =>
     void (async (event: React.FormEvent<CandidatesVotesFormElement>) => {
       event.preventDefault();
 
-      if (errors.length === 0 && warnings.length > 0) {
+      const values = getValues();
+      if (values.candidate_votes.some((candidate) => candidate.votes > 0) && values.total === 0) {
+        setMissingTotalError(true);
+        document.getElementById("total")?.focus();
```

Comment:
I think a ref would be nicer here.

Reply:
oh hmm, nvm, that would require implementing a forwardRef which is kinda iffy.

Reply:
Yeah I thought the same